scenes/startup.txt

```diff
@@ -102,6 +102,9 @@
 *create dragon_1_hit_points 100
 *create dragon_2_name "Hobbes"
 *create dragon_2_hit_points 150
+*create dragon_3_name "Stebbing"
+*create dragon_3_hit_points 100
+*create dragon_max 3
```

Comment:
If we're going to use _max, we should define _count as well (see: https://github.com/ChoicescriptIDE/cslib/wiki/Arrays), for consistency (even if it's not used).